doc/source/docs/user_guide/geometric_manipulations.rst

```diff
@@ -12,63 +12,63 @@ Constructive Methods
 
 .. method:: GeoSeries.buffer(distance, resolution=16)
```

Comment:
Can we somehow link these to their API reference? This page predates the new API documentation and we should update it to play better with the rest.

Reply:
I looked into the rst [documentation](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html), and tried different linking combinations  (similar to the new changes), but I was not able to make it link back to the API reference unfortunately.

Reply:
Hmm, never mind. We'll deal with this page later.